index.js

```diff
@@ -82,6 +85,16 @@ async function underPressure (fastify, opts) {
 
   fastify.addHook('onRequest', onRequest)
 
+  function mapExposeStatusRoute (opts) {
+    if (!opts) {
+      return false
+    }
+    const route = typeof opts === 'string' ? opts : '/status'
```

Comment:
These lines are quite hard to read
We could simplify it and remove the duplicated `/status` in the `url: opts.exposeStatusRoute.route || '/status',` line

```
    if (typeof opts === 'string') {
      return { route: opts }
    }

    return Object.assign({ route: '/status' }, opts)
```

```

Reply:
Code simplified